src/pages/AddBoxForm/AddBoxForm.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ChakraProvider } from '@chakra-ui/react';
+import BoxForm from '../../components/BoxForm/BoxForm';
+import './AddBoxForm.css';
+
+function AddBoxForm() {
```

Comment:
Change component to arrow function to follow ES6 conventions and so that we can keep things consistent throughout